org.eclipse.wildwebdeveloper.feature/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.eclipse.wildwebdeveloper</groupId>
 		<artifactId>parent</artifactId>
-		<version>0.2.0-SNAPSHOT</version>
+		<version>0.2.1-SNAPSHOT</version>
```

Comment:
Parent version doesn't have to change since we didn't change any build-related stuff.